### ISSUE TITLE
meta: New backends

### ISSUE THREAD

**Issue by lostmygithubaccount** [ibis-project/ibis#8035]
**Acceptance criteria**: Adding new backends for the core Ibis team is not a priority, but we will support new backends from the community or organizations partnering with the Ibis project. This meta-issue for the roadmap is to track new backends for Q1 2024.

Adding new backends for popular data platforms will be a priority after #7909 and related work to stabilize the backend interface.

TODOs:

- [x] https://github.com/ibis-project/ibis/issues/8037
- [x] https://github.com/ibis-project/ibis/issues/8036
- [x] https://github.com/ibis-project/ibis/issues/8038

**Comment by lostmygithubaccount** [comment]
**Weekly Update**:
2024-01-25

- RisingWave backend is still in progress
- no other new backends planned at this time

**Comment by lostmygithubaccount** [comment]
no update or new backends planned this quarter; will update if that changes

**Comment by lostmygithubaccount** [comment]
chDB in progress #8497

**Comment by lostmygithubaccount** [comment]
closing out from last quarter
